src/CabanaPD_Solver.hpp

```diff
@@ -258,16 +271,23 @@ class SolverElastic
             // Integrate - velocity Verlet first half.
```

Comment:
Can you remind me what gets updated during "velocity Verlet first half" and "velocity Verlet second half"?

Reply:
I think we need to keep this review focused on the heat transfer